Assets/_TransmissionGGJ/Teams/Francois/Scripts/ScreenClickPlayerMove.cs

```diff
@@ -37,9 +44,54 @@ void Update()
         {
             Debug.Log(m_CitizenToFollow.transform.position);
             m_AIcontroller.SetTarget(m_CitizenToFollow.position);
+            if (CheckIfArrived())
+            //if (InitiateConversation(m_CitizenToFollow.GetComponent<CitizenAgent>()))
+            {
+                m_citizenAgent.StartConversationWith(m_recipientCitizenAgent, 20, _initiator: true);
```

Comment:
Conversation are back to 20s? Also, no more checking if the recipient is busy, dead or available to talk to.